x11-libs/libva-intel-media-driver/libva-intel-media-driver-9999.ebuild

```diff
@@ -22,19 +22,40 @@ HOMEPAGE="https://github.com/intel/media-driver"
 
 LICENSE="MIT BSD"
 SLOT="0"
-IUSE=""
+IUSE="+custom-cflags latest-cxx set-as-default test X"
 
-DEPEND=">=media-libs/gmmlib-${PV}
-	>=x11-libs/libva-2.8.0
+RESTRICT="!test? ( test )"
+
+PATCHES=( "${FILESDIR}"/${PN}-20.2.0_x11_optional.patch
+          "${FILESDIR}"/${PN}-20.4.5_custom_cflags.patch
+          "${FILESDIR}"/${PN}-20.4.5_tesing_in_src_test.patch
+        )
+
+DEPEND=">=media-libs/gmmlib-20.4.1
+	>=x11-libs/libva-2.10.0[X?]
 	>=x11-libs/libpciaccess-0.13.1-r1:=
 "
 RDEPEND="${DEPEND}"
 
 src_configure() {
 	local mycmakeargs=(
 		-DMEDIA_BUILD_FATAL_WARNINGS=OFF
-		-DMEDIA_RUN_TEST_SUITE=OFF
+		-DMEDIA_RUN_TEST_SUITE=$(usex test)
+		-DBUILD_TYPE=release
+		-DPLATFORM=linux
+		-DUSE_X11=$(usex X)
+		-DLATEST_CPP_NEEDED=$(usex latest-cxx)
```

Comment:
Sorry, one more question - is this needed, or should we just pick on/off?

Reply:
OK, selected `latest C++`. Works fine.